iothub_client/inc/iothub_module_client.h

```diff
@@ -249,6 +252,46 @@ extern "C"
     */
     MOCKABLE_FUNCTION(, IOTHUB_CLIENT_RESULT, IoTHubModuleClient_GetTwinAsync, IOTHUB_MODULE_CLIENT_HANDLE, iotHubModuleClientHandle, IOTHUB_CLIENT_DEVICE_TWIN_CALLBACK, moduleTwinCallback, void*, userContextCallback);
 
+    /**
+    * @brief    This API provides a way to retrieve the module Twin desired properties on-demand.
+    *
+    * @param    iotHubModuleClientHandle    The handle created by a call to the create function.
+    * @param    twinRequestOptions          Pointer to the GET twin request options struct. Can only
+    *                                       be used when requesting desired or reported sections.
+    * @param    moduleTwinDesiredCallback   The callback invoked to provide the Module Twin desired
+    *                                       properties once its retrieval is completed by the client.
+    *                                       If any failures occur, the callback is invoked passing
+    *                                       @c NULL as payLoad and zero as size.
```

Comment:
can probably do a find all and replace on this guy
```suggestion
    *                                       @c NULL as payload and zero as size.
```